src/genie/libs/parser/iosxe/show_lisp.py

```diff
@@ -3412,3 +3412,168 @@ def filter_lines(raw_output, remove_lines):
         return lisp_site_dict
 
 
+
+
+# ================================================
+# Schema for:
+#  * 'show lisp eid-table vrf User ipv4 map-cache'
+# ================================================
+class ShowLispEidTableVrfUserIpv4MapCacheSchema(MetaParser):
+    """Schema for show lisp eid-table vrf User ipv4 map-cache."""
```

Comment:
User is a vrf name, so it can be passed as an argument?
```
show lisp eid-table vrf {vrf} ipv4 map-cache

Reply:
Updated